Automattic/wp-calypso
# Launchpad: Renamed keep-building tasklist

Related to https://github.com/Automattic/jetpack/pull/31905

DO NOT MERGE THIS PR until https://github.com/Automattic/jetpack/pull/31905 has been shipped to both WordPress.com Simple and Atomic sites.

## Proposed Changes

* This PR renames the `keep-building` tasklist to `intent-write`, to follow the pattern.

## Testing Instructions



**Test on a WP.com Simple site:**

* Apply D116143-code to the sandbox
* Apply https://github.com/Automattic/jetpack/pull/31905 to the sandbox
* Ensure you're working from this diff, either from calypso.live or locally
* Create a new site (`/start`) with `build` intent by selecting "Promote myself or business" during onboarding.
* Launch the site
* Make sure the Launchpad in Customer Home still shows and works:
![image](https://github.com/Automattic/wp-calypso/assets/3801502/5e573964-4d55-4308-8a26-15c08ef4fb4e)

**Test on an Atomic site:**

* Apply D116143-code to the sandbox
* Create a new site on the Business plan from `/start` with the Build intent and launch it.
* Install and activate a free plugin to transfer the site to Atomic infrastructure.
* SSH into your Atomic site and add `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );` to your Atomic site's `wp-config.php`
* Apply https://github.com/Automattic/jetpack/pull/31905 to your Atomic site via the [Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/); in the plugin management screen, go to WordPress.com Features and search for this branch, then activate it.
* Ensure you're browsing the UI with this diff, either from calypso.live or locally.
* Visit `/home/yoursiteslug` and you should still see the Build checklist; you should be able to complete the checklist tasks.
